locales/sw/json.json

```diff
@@ -77,13 +77,13 @@
     "Apply Discount": "Tumia punguzo",
     "Applying": "Inatuma",
     "April": "Aprili",
-    "Are you sure you want to attempt to pay :amount?": "Are you sure you want to attempt to pay :amount?",
+    "Are you sure you want to attempt to pay :amount?": "Je, una uhakika unataka kujaribu kulipa :amount?",
     "Are you sure you want to cancel your subscription?": "Je, una uhakika unataka kughairi usajili wako?",
     "Are you sure you want to delete the selected resources?": "Je, una uhakika unataka kufuta rasilimali zilizochaguliwa?",
     "Are you sure you want to delete this announcement?": "Je, una uhakika unataka kufuta tangazo hili?",
     "Are you sure you want to delete this file?": "Je, una uhakika unataka kufuta faili hii?",
     "Are you sure you want to delete this notification?": "Je, una uhakika unataka kufuta arifa hii?",
-    "Are you sure you want to delete this payment method?": "Are you sure you want to delete this payment method?",
+    "Are you sure you want to delete this payment method?": "Je, una uhakika unataka kufuta njia hii ya kulipa",
```

Comment:
Surely there shouldn't be a question mark here?

Reply:
Oh, I have fixed it